openff/toolkit/topology/topology.py

```diff
@@ -1021,10 +1019,8 @@ def chemical_environment_matches(
         """
 
         # Render the query to a SMARTS string
-        if type(query) is str:
+        if isinstance(query, str):
             smarts = query
-        elif type(query) is ChemicalEnvironment:
-            smarts = query.as_smarts()
```

Comment:
There is no `ChemicalEnvironment.as_smarts` or any such method elsewhere in this codebase

```shell
$ git checkout 0.11.4 && grep -r "as_smarts" openff
HEAD is now at 516048f4 0.11.4 release prep
Binary file openff/toolkit/topology/__pycache__/topology.cpython-310.pyc matches
openff/toolkit/topology/topology.py:            Query will internally be resolved to SMARTS using ``query.as_smarts()`` if it has an ``.as_smarts`` method.
openff/toolkit/topology/topology.py:            smarts = query.as_smarts()
```

https://app.codecov.io/gh/openforcefield/openff-toolkit/blob/main/openff/toolkit/topology/topology.py

Reply:
Ah, good find. Yeah, we can drop those lines then!